fairship.sh

```diff
@@ -25,7 +26,7 @@ incremental_recipe: |
   rsync -a $INSTALLROOT/*/*.h $INSTALLROOT/include
   #Get the current git hash
   cd $SOURCEDIR
-  FAIRSHIP_HASH=$(git rev-parse HEAD)
```

Comment:
Don't understand why removing FAIRSHIP_HASH against SNDSW. SNDSW/SND should not appear at all in this directory, I would have assumed.

Reply:
Indeed, that was a mistake. Fixed now.